### ISSUE TITLE
Not sure why this is happening

### ISSUE THREAD

**Issue by YouKnwNothing** [Xemorr/StopRenaming#1]
![image](https://user-images.githubusercontent.com/92229785/189464985-4d471912-3e8f-4c97-8500-9ffefa218982.png)

**Comment by Xemorr** [comment]
Hi, YouKnwNothing, is this the first error in the console related to StopRenaming?

**Comment by YouKnwNothing** [comment]
Yeah I just updated my paper.jar and then this happened. It was working fine before that. Im on 1.19.2

**Comment by Xemorr** [comment]
Can you scroll up in console, I don't think this is the first error. Please scroll up to the point at which the plugin disables itself

**Comment by YouKnwNothing** [comment]
started working again.
